python/lsst/daf/persistence/butler.py

```diff
@@ -39,15 +39,9 @@
 import lsst.pex.logging as pexLog
 import lsst.pex.policy as pexPolicy
 from lsst.daf.persistence import StorageList, LogicalLocation, ReadProxy, ButlerSubset, ButlerDataRef, \
```

Comment:
It seems to depend on coupling. IE you can run into problem if A comes before B in the __init__.py file and A imports B. But as long as the classes are coupled properly it works fine.